crates/wasmtime/src/runtime/component/store.rs

```diff
@@ -9,7 +9,7 @@ use wasmtime_environ::PrimaryMap;
 
 #[derive(Default)]
 pub struct ComponentStoreData {
-    instances: PrimaryMap<ComponentInstanceId, Option<OwnedComponentInstance>>,
+    pub(crate) instances: PrimaryMap<ComponentInstanceId, Option<OwnedComponentInstance>>,
```

Comment:
I'm wary of making this public to the whole crate as it generally makes it harder to maintain abstraction boundaries. Would it be possible to encapsulate whatever's   needed within this file instead?